worker/include/RTC/SeqManager.hpp

```diff
@@ -7,11 +7,11 @@
 
 namespace RTC
 {
-	template<typename T>
+	template<typename T, unsigned int N = 0>
```

Comment:
Using `uint8_t` type looks more concise IMHO and it's more than enough space to hold all the possible values.

```suggestion
	template<typename T, uint8_t N = 0>
```

Reply:
Also please add a little comment indicating that `N` represents the number of bits to be used in `T`.